src/components/Adds/Adds.js

```diff
@@ -0,0 +1,112 @@
+import TextField from '@mui/material/TextField';
+import Button from '@mui/material/Button';
+
+export default function Adds() {
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    const response = await fetch('http://127.0.0.1:3000/cars/create', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({
+        model: e.target.model.value,
+        color: e.target.color.value,
+        image: e.target.image.value,
+        range: e.target.range.value,
+        motor_type: e.target.motor_type.value,
+        acceleration_time: e.target.acceleration_time.value,
+        booking_price: e.target.booking_price.value,
+        booking_duration: e.target.booking_duration.value,
+      }),
+    });
+    response.json();
+
+    if (response.status === 201) {
+      window.location.reload();
+    }
+
+    return response;
+  };
+
+  return (
+    <div>
+      <form
+        id="form"
```

Comment:
[OPTIONAL] Please consider adding a header to your form page, like 'Add a new model' or 'Add a new car'. This will improve the user experience 😉.

Reply:
Done